vllm/model_executor/model_loader.py

```diff
@@ -38,6 +38,16 @@
     "YiForCausalLM": YiForCausalLM,
 }
 
+# Models to be disabled in ROCm
+_ROCM_DISABLED_MODELS = [
+    # ROCm's flash attention does not support sliding window attention
+    # in models such as Mistral
+    "MistralForCausalLM",
+]
+if torch.version.hip:
```

Comment:
Instead of directly using this, can we define a function like `is_hip()` in `vllm/utils.py`?